### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by TomDLT**
Just a quick pass on the README.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,26 +3,29 @@ Benchmark repository for optimization
...
+BenchOpt is implemented in Python, and can run algorithms 
+written in **many programming languages**
+(`example <https://benchopt.github.io/auto_examples/plot_run_benchmark_python_R.html>`_).
+So far, BenchOpt has been tested with `Python <https://www.python.org/>`_,
 `R <https://www.r-project.org/>`_, `Julia <https://julialang.org/>`_
-and compiled binaries written in C/C++ available via a terminal
-command. If it can be installed via
-`conda <https://docs.conda.io/en/latest/>`_ it should just work!
+and `C/C++ <https://isocpp.org/>`_ (compiled binaries with a command line interface).
+Most programs available via
```
Comment (mathurinm) [inline README.rst:15]:
```suggestion
Programs available via
```

File: README.rst [hunk 2]
```diff
@@ -32,36 +35,33 @@ logistic regression:
...
+The command line tool to run the benchmarks can be installed through `pip`.
+To get the **latest release**, use:
 
 .. code-block::
 
-    $ pip install benchopt
+    pip install benchopt
 
-And to get the **latest development version**, you can use:
+To get the **latest development version**, you can use:
```
Comment (mathurinm) [inline README.rst:53]:
```suggestion
To get the **latest development version**, use:
```

File: README.rst [hunk 3]
```diff
@@ -3,26 +3,29 @@ Benchmark repository for optimization
...
+BenchOpt is a benchmarking suite for optimization algorithms.
+It is built for simplicity, transparency, and reproducibility.
 
-BenchOpt is written in Python but it is available with
-`many programming languages <https://benchopt.github.io/auto_examples/plot_run_benchmark_python_R.html>`_.
-So far it has been tested with `Python <https://www.python.org/>`_,
+BenchOpt is implemented in Python, and can run algorithms 
+written in **many programming languages**
+(`example <https://benchopt.github.io/auto_examples/plot_run_benchmark_python_R.html>`_).
+So far, BenchOpt has been tested with `Python <https://www.python.org/>`_,
```
Comment (mathurinm) [inline README.rst:12]:
```suggestion
So far, Benchopt has been tested with `Python <https://www.python.org/>`_,
```

File: README.rst [hunk 4]
```diff
@@ -3,26 +3,29 @@ Benchmark repository for optimization
...
 
-BenchOpt is a package to simplify, make more transparent and
-more reproducible the comparisons of optimization algorithms.
+BenchOpt is a benchmarking suite for optimization algorithms.
+It is built for simplicity, transparency, and reproducibility.
 
-BenchOpt is written in Python but it is available with
-`many programming languages <https://benchopt.github.io/auto_examples/plot_run_benchmark_python_R.html>`_.
-So far it has been tested with `Python <https://www.python.org/>`_,
+BenchOpt is implemented in Python, and can run algorithms
```
Comment (mathurinm) [inline README.rst:9]:
```suggestion
Benchopt is implemented in Python, and can run algorithms 
```

**Review by tomMoral — APPROVED** [review]
LGTM, thx @TomDLT

**Pull request merged**